test/behavior/floatop.zig

```diff
@@ -697,3 +697,38 @@ test "f128 at compile time is lossy" {
 
     try expect(@as(f128, 10384593717069655257060992658440192.0) + 1 == 10384593717069655257060992658440192.0);
 }
+
+test "f16 literals" {
+    if (builtin.zig_backend != .stage2_c) return error.SkipZigTest; // TODO
```

Comment:
Why are this test and the next being skipped for the C backend? I would assume them to be passing after your changes.

Reply:
You're right, only the f128 tests are not passing, I'll update again.

Reply:
``` 
stage2/bin/zig test test/behavior.zig -I test -target aarch64-linux --test-cmd qemu-aarch64 --test-cmd-bin
test/behavior/floatop.zig:701:1: error: TODO implement const of type 'f16'
```
Actually it's not passing, although for a different reason than f128. So I think it should still be skipped for backend != c.